src/main/java/com/portatlas/Server.java

```diff
@@ -1,11 +1,72 @@
 package com.portatlas;
 
+import com.portatlas.request.Request;
+import com.portatlas.request.RequestParser;
+import com.portatlas.response.*;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+
 public class Server {
 
-    public static void main(String[] args) {
-        ArgParser argParser = new ArgParser();
+    public static Router router = new Router();
+    public static ArgParser argParser = new ArgParser();
+    public static Response response;
+
+    public static void main(String[] args) throws IOException {
+        ServerSocket server = configureServer(args);
+        startServer(server);
+    }
+
+    public static ServerSocket configureServer(String[] args) throws IOException {
         argParser.parseArgs(args);
         argParser.printArgs(argParser.port, argParser.dir);
+
+        Router router = addRoutes();
+
+        return new ServerSocket(argParser.port);
+    }
+
+    public static Router addRoutes() {
+        router.addRoute(new Request("GET", "/" , "HTTP/1.1"), OkResponse.run());
+        router.addRoute(new Request("HEAD", "/" , "HTTP/1.1"), OkResponse.run());
+        router.addRoute(new Request("OPTIONS", "/method_options" , "HTTP/1.1"), OptionResponse.run("Allow", "GET,HEAD,POST,OPTIONS,PUT"));
+        router.addRoute(new Request("OPTIONS", "/method_options2" , "HTTP/1.1"), OptionResponse.run("Allow", "GET,OPTIONS"));
+        router.addRoute(new Request("POST", "/form" , "HTTP/1.1"), OkResponse.run());
+        router.addRoute(new Request("PUT", "/form" , "HTTP/1.1"), OkResponse.run());
```

Comment:
Could you try creating constants for the strings here? I think "HTTP/1.1" and the HTTP methods like "PUT" can also be a constant. Maybe look into enums as well for the HTTP methods